docs-java/spring-ai/openai.mdx

```diff
@@ -50,3 +50,98 @@ float[] embeddings = embeddingModel.embed(texts);
 ```
 
 Please find [an example in our Spring Boot application](https://github.com/SAP/ai-sdk-java/tree/main/sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/SpringAiOpenAiService.java).
+
+## Chat Completion
+
+The OpenAI client is integrated in Spring AI classes:
+
+```java
+ChatModel client = new OpenAiChatModel(OpenAiClient.forModel(OpenAiModel.TEXT_EMBEDDING_3_SMALL));
+OpenAiChatOptions opts = new OpenAiChatOptions();
```

Comment:
For better readability in your code blocks, please extract the client to a dedicated variable.

```suggestion
OpenAiClient client = OpenAiClient.forModel(OpenAiModel.TEXT_EMBEDDING_3_SMALL);
ChatModel client = new OpenAiChatModel(client);
OpenAiChatOptions opts = new OpenAiChatOptions();
```